rfc6962-bis.xml

```diff
@@ -801,15 +801,16 @@ signatures.
           The leaves of the Merkle Tree are the leaf hashes of the corresponding <spanx style="verb">MerkleTreeLeaf</spanx> structures. Note that <xref target="mht">leaf hashes</xref> are calculated as HASH(0x00 || MerkleTreeLeaf).
         </t>
       </section>
-      <section title="Signed Tree Head" anchor="STH">
+      <section title="Signed Tree Head (STH)" anchor="STH">
         <t>
           Periodically the log SHOULD sign the corresponding tree hash and tree information (see the corresponding Signed Tree Head client message in <xref target="fetch_sth"/>).
         </t>
-        <figure>
-          <preamble>
-            The signature for that data is structured as follows:
-          </preamble>
-          <artwork>
+        <t>
+          Each log MUST produce on demand a Signed Tree Head that is no older than the Maximum Merge Delay. However, Signed Tree Heads could be used to mark individual clients (by producing a new one for each query), so logs MUST NOT produce them more frequently than is declared in their metadata. In general, there is no need to produce a new Signed Tree Head unless there are new entries in the log, however, in the unlikely event that it receives no new submissions during an MMD period, the log SHALL sign the same Merkle Tree Hash with a fresh timestamp.
```

Comment:
That's true, but that text wasn't introduced by this change.  Let's tackle that in a new PR.